### PR TITLE
Fix bug in the method #toc_changes?

### PR DESCRIPTION
**Pull request by mr-dxdy**

### CODE REVIEW

**Comment by dblock** [comment]
Thanks!

Why didn't Travis trigger a build here? Everything seems to be setup correctly . ... Either way the CHANGELOG needs an update, please.

**Comment by dblock** [comment]
Interesting...

screen shot 2018-03-21 at 12 24 33 pm

I'll e-mail Travis support.

**Comment by mr-dxdy** [comment]
hmm, rubocop failed.

**Comment by dblock** [comment]
Run `rubocop -a ; rubocop --auto-gen-config`, that's my usual workflow.

**Comment by mr-dxdy** [comment]
I found in .travis.yml line `rvm: 2.4.1` and i decided that minimum support ruby version is 2.4.1. Correctly?

**Comment by dblock** [comment]
Oh I see what's happening. Just lock down the version of Rubocop to 0.41.0 in .gemspec and it will work? We should separately upgrade it.

**Comment by mr-dxdy** [comment]
Thanks! But i dont like that .rubocop_todo.yml was created by rubocop 0.51.0. [Link](https://github.com/dblock/danger-toc/blob/master/.rubocop_todo.yml#L3).

```
Warning: unrecognized cop Naming/UncommunicativeMethodParamName found in /home/travis/build/dblock/danger-toc/.rubocop_todo.yml
Warning: unrecognized cop Layout/IndentHeredoc found in /home/travis/build/dblock/danger-toc/.rubocop_todo.yml
Warning: unrecognized cop Metrics/BlockLength found in /home/travis/build/dblock/danger-toc/.rubocop_todo.yml
Warning: unrecognized cop Metrics/BlockLength found in /home/travis/build/dblock/danger-toc/.rubocop.yml
Warning: unrecognized cop Naming/UncommunicativeMethodParamName found in /home/travis/build/dblock/danger-toc/.rubocop.yml
Warning: unrecognized cop Layout/IndentHeredoc found in /home/travis/build/dblock/danger-toc/.rubocop.yml
```

**Comment by dblock** [comment]
That's why you re-generate one with whatever locked down version is, `rubocop -a ; rubocop --auto-gen-config` should do it, no?

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (dblock) [inline Gemfile]: We don't want this one.

**Comment by mr-dxdy** [comment]
sorry, I didn't want to lose your custom settings in .rubocop_todo.yml

**Comment by dblock** [comment]
Merged, thanks!

Want to help out with this gem @mr-dxdy? Happy to add you to contributors. Maybe you can cut the next release (what's your rubygems email)?

**Comment by mr-dxdy** [comment]
Yes, ready to help in the development of the gem. Next of version of this gem i wanted to release after a successful closing a issue [1753](https://github.com/ruby-grape/grape/pull/1753)

**Comment by dblock** [comment]
What's your Rubygems email?

**Comment by mr-dxdy** [comment]
dxdy@bk.ru

**Comment by dblock** [comment]
Added you to Rubygems @mr-dxdy.

**Pull request merged**